util/addDataFromZib/addDataFromZib.xsl

```diff
@@ -20,6 +20,146 @@
         </xsl:copy>
     </xsl:template>
     
+    <xsl:template match="f:StructureDefinition">
+        <xsl:copy>
+            <xsl:variable name="id" select="f:id/@value"/>
+            <xsl:apply-templates select="f:id | f:meta | f:implicitRules | f:language | f:text | f:contained | f:extension | f:modifierExtension"/>
+            
+            <!-- Add or modify URL -->
+            <xsl:choose>
+                <xsl:when test="(not(f:url) or starts-with(f:url/@value, 'http://example.org/')  or starts-with(f:url/@value, 'https://example.org/')) and (starts-with($id, 'zib-') or starts-with($id, 'nl-core-'))">
+                    <url value="http://nictiz.nl/fhir/StructureDefinition/{$id}"/>
+                </xsl:when>
+                <xsl:otherwise>
+                    <xsl:apply-templates select="f:url"/>
+                </xsl:otherwise>
+            </xsl:choose>
+            
+            <xsl:apply-templates select="f:identifier | f:version"/>
+            
+            <!-- Add or modify name, title, status -->
+            <xsl:choose>
+                <xsl:when test="(not(f:url) or starts-with(f:name/@value, 'My')) and (starts-with($id, 'zib-') or starts-with($id, 'nl-core-'))">
```

Comment:
Why just "zib-" and "nl-core-"?

Reply:
Behoudend coderen - Dus met rust laten als er niet per se iets mee moet gebeuren. Maar in dit geval zou dat inderdaad wel weg kunnen